Directory.Solution.Props

```diff
@@ -0,0 +1,3 @@
+<Project>
+    <Import Project="Directory.Build.Props" />
```

Comment:
Prefer lower-case file extension instead of Title-case.

Reply:
The casing matters cross platform.
I incorrectly remembered what the xplat behavior is.
`props` and `targets` are what MSBuild looks for. 

Nice catch.